### DISCUSSION TITLE
Trying to Home with UNO Shild V3

### DISCUSSION THREAD

**Discussion by manolocs** [winder/Universal-G-Code-Sender#1976] [Help]
Hello, new in GRBL, following the UGS Setup Wizard, all working until the Homing page, when click Try Homing the "Y axis" home return and stop again at the end stop swt, the "X axis" goes to the swt and stops does not retract like the "Y axis" and the Z does not move at all.Any idea what Am I doing wrong? 

Using UGS V2.0.12
Windows 10
Arduino UNO original
Arduino Shield V3

>>> $X
Grbl 1.1h ['$' for help]
>>> $$
$0 = 10    (Step pulse time, microseconds)
$1 = 25    (Step idle delay, milliseconds)
$2 = 0    (Step pulse invert, mask)
$3 = 0    (Step direction invert, mask)
$4 = 0    (Invert step enable pin, boolean)
$5 = 0    (Invert limit pins, boolean)
$6 = 0    (Invert probe pin, boolean)
$10 = 3    (Status report options, mask)
$11 = 0.010    (Junction deviation, millimeters)
$12 = 0.002    (Arc tolerance, millimeters)
$13 = 0    (Report in inches, boolean)
$20 = 0    (Soft limits enable, boolean)
$21 = 1    (Hard limits enable, boolean)
$22 = 1    (Homing cycle enable, boolean)
$23 = 3    (Homing direction invert, mask)
$24 = 25.000    (Homing locate feed rate, mm/min)
$25 = 500.000    (Homing search seek rate, mm/min)
$26 = 250    (Homing switch debounce delay, milliseconds)
$27 = 1.000    (Homing switch pull-off distance, millimeters)
$30 = 1000    (Maximum spindle speed, RPM)
$31 = 0    (Minimum spindle speed, RPM)
$32 = 0    (Laser-mode enable, boolean)
$100 = 842.077    (X-axis travel resolution, step/mm)
$101 = 795.453    (Y-axis travel resolution, step/mm)
$102 = 3254.530    (Z-axis travel resolution, step/mm)
$110 = 500.000    (X-axis maximum rate, mm/min)
$111 = 500.000    (Y-axis maximum rate, mm/min)
$112 = 500.000    (Z-axis maximum rate, mm/min)
$120 = 10.000    (X-axis acceleration, mm/sec^2)
$121 = 10.000    (Y-axis acceleration, mm/sec^2)
$122 = 10.000    (Z-axis acceleration, mm/sec^2)
$130 = 200.000    (X-axis maximum travel, millimeters)
$131 = 200.000    (Y-axis maximum travel, millimeters)
$132 = 200.000    (Z-axis maximum travel, millimeters)
ok
>>> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M9 T0 F0 S0]
ok
>>> $X
[MSG:Caution: Unlocked]
ok
>>> $H
ok
An unexpected error was detected: (ALARM:1) Hard limit has been triggered. Machine position is likely lost due to sudden halt. Re-homing is highly recommended.
[MSG:Reset to continue]

**Comment by breiler** [comment]
Normally the Z-axis will be homed first, after that both X and Y will be homed together. So I'm guessing that something is wrong with your wiring.

**Comment by manolocs** [comment]
Thanks for the reply,  I will check but all axis are working fine I tested before I enable the homing and in the last page of the setup wizard they all are working as it should.
